droplets.go

```diff
@@ -218,37 +221,46 @@ func (d DropletCreateSSHKey) MarshalJSON() ([]byte, error) {
+	Region            string                     `json:"region"`
+	Size              string                     `json:"size"`
+	Image             DropletCreateImage         `json:"image"`
+	SSHKeys           []DropletCreateSSHKey      `json:"ssh_keys"`
+	Backups           bool                       `json:"backups"`
+	IPv6              bool                       `json:"ipv6"`
+	PrivateNetworking bool                       `json:"private_networking"`
+	Monitoring        bool                       `json:"monitoring"`
+	UserData          string                     `json:"user_data,omitempty"`
+	Volumes           []DropletCreateVolume      `json:"volumes,omitempty"`
+	Tags              []string                   `json:"tags"`
+	VPCUUID           string                     `json:"vpc_uuid,omitempty"`
+	WithDropletAgent  *bool                      `json:"with_droplet_agent,omitempty"`
+	BackupPolicy      *BackupPolicyCreateRequest `json:"backup_policy,omitempty"`
 }
 
 // DropletMultiCreateRequest is a request to create multiple Droplets.
 type DropletMultiCreateRequest struct {
-	Names             []string              `json:"names"`
-	Region            string                `json:"region"`
-	Size              string                `json:"size"`
-	Image             DropletCreateImage    `json:"image"`
-	SSHKeys           []DropletCreateSSHKey `json:"ssh_keys"`
-	Backups           bool                  `json:"backups"`
-	IPv6              bool                  `json:"ipv6"`
-	PrivateNetworking bool                  `json:"private_networking"`
-	Monitoring        bool                  `json:"monitoring"`
-	UserData          string                `json:"user_data,omitempty"`
-	Tags              []string              `json:"tags"`
-	VPCUUID           string                `json:"vpc_uuid,omitempty"`
-	WithDropletAgent  *bool                 `json:"with_droplet_agent,omitempty"`
+	Names             []string                   `json:"names"`
+	Region            string                     `json:"region"`
+	Size              string                     `json:"size"`
+	Image             DropletCreateImage         `json:"image"`
+	SSHKeys           []DropletCreateSSHKey      `json:"ssh_keys"`
+	Backups           bool                       `json:"backups"`
+	IPv6              bool                       `json:"ipv6"`
+	PrivateNetworking bool                       `json:"private_networking"`
+	Monitoring        bool                       `json:"monitoring"`
+	UserData          string                     `json:"user_data,omitempty"`
+	Tags              []string                   `json:"tags"`
+	VPCUUID           string                     `json:"vpc_uuid,omitempty"`
+	WithDropletAgent  *bool                      `json:"with_droplet_agent,omitempty"`
+	BackupPolicy      *BackupPolicyCreateRequest `json:"backup_policy,omitempty"`
+}
+
+// BackupPolicyCreateRequest defines the backup policy when creating a Droplet.
+type BackupPolicyCreateRequest struct {
```

Comment:
I think we'll also need this for the update method. So maybe just `DropletBackupPolicyRequest`? Namespaced without `Create`?

Reply:
:+1: Makes sense. Thanks. Updated in [6ae90e3](https://github.com/digitalocean/godo/pull/749/commits/6ae90e39309308297e4800813585229117be3ddd)

@andrewsomething , could you advise if I need to add a dedicated droplet action for Update such as - `ChangeBackupPolicy(context.Context, int) (*Action, *Response, error)`

I'm asking because backup policies already can be changed with `EnableBackups(context.Context, int) (*Action, *Response, error)`

[API doc](https://docs.digitalocean.com/reference/api/api-reference/#operation/dropletActions_post)

Reply:
Ya, I think we'll need:

```
ChangeBackupPolicy(context.Context, int, DropletBackupPolicyRequest) (*Action, *Response, error)
```

Can you also verify that a `change_backup_policy` request works to enable backups on a Droplet that does not have them currently enabled? I believe it should, but if not we'll need something like `EnableBackupsWithPolicy` so we don't break the signature of the existing `EnableBackup` method.

Reply:
Yep, that's a good case, I'll check :+1:

Reply:
Andrew, I'd better discuss [e8765e3](https://github.com/digitalocean/godo/pull/749/commits/e8765e333c2bccb90855ddc68f241c5275e94d15) on a call. This is the best I could do. 
Would be great to discuss other ways to implement it without triggering `Droplet Issue` notification in a console UI.

More info:
- I can't use Droplets.GetBackupPolicy in DropletActions to check if droplet backups are enabled or not. I believe it can't be done without code changes that will increase code complexity.
- If backups disable action applied to the droplet which is already disabled - we see a UI message "Droplet Issue" in console.
- If backups enable action applied to the droplet which is already eabled - we see a UI message "Droplet Issue" in console.

Reply:
Updated in [5e46424](https://github.com/digitalocean/godo/pull/749/commits/5e46424ab1d28dfc104c32e37e16791fc0c47c7e)
Droplet backups have to be enabled.